### PR TITLE
Add Master's weekly CI build

### PR DESCRIPTION
**Pull request by kdmccormick**
@edx/masters-neem 
https://openedx.atlassian.net/browse/EDUCATOR-4519

**Do not merge yet!**
Blocked by https://github.com/edx/edx-internal/pull/1019

**After merging**
Update https://openedx.atlassian.net/wiki/spaces/MS/pages/1028489324/Master+s+Integration+Environment

### CODE REVIEW

File: devops/jobs/CreateSandboxCI.groovy [hunk 1]
```diff
@@ -52,17 +52,70 @@ class CreateSandboxCI {
...
+                // https://github.com/edx/registrar/blob/master/openedx.yaml
+                } else if (type == 'MastersWeekly') {
+                  predefinedProp('dns_name','masters-weekly')
+                  predefinedProp('name_tag','masters-weekly')
+
+                  // Define to 30 because:
+                  // * This will be re-built every week anyway
+                  // * We want real Masters integration boxes to be able to use
+                  //   these parameters as defaults.
+                  predefinedProp('sandbox_life','30')
```
Comment (schenedx) [inline devops/jobs/CreateSandboxCI.groovy]: Why not define this to 7 then, since it will get wiped in 7 days without intervention?

Comment (kdmccormick) [reply devops/jobs/CreateSandboxCI.groovy]: Do you want the `masters-weekly` box or the intergration environments to be wiped in 7 days without intervention?

Comment (schenedx) [reply devops/jobs/CreateSandboxCI.groovy]: I think that's fine for now, because if the next rebuild succeeds, we still have the environment.

Comment (schenedx) [reply devops/jobs/CreateSandboxCI.groovy]: We synced offline. And I am fine with having this parameter to be 30

**Pull request merged**